tests/client/amo/containers/TestHome.js

```diff
@@ -4,13 +4,14 @@ import {
   renderIntoDocument,
 } from 'react-addons-test-utils';
 
+import { getFakeI18nInst } from 'tests/client/helpers';
 import Home from 'amo/containers/Home';
 
 
 describe('Home', () => {
   it('renders a heading', () => {
-    const root = renderIntoDocument(<Home />);
+    const root = renderIntoDocument(<Home i18n={getFakeI18nInst()} />);
     const rootNode = findDOMNode(root);
-    assert.include(rootNode.querySelector('h1').textContent, 'AMO Home Page');
+    assert.include(rootNode.querySelectorAll('h2')[0].textContent, 'What do you want Firefox to do?');
```

Comment:
Could leave this as `querySelector` if you're just checking the first one. Doing `.map((el) => el.textContent)` and checking `['What do you want Firefox to do?', 'How do you want Firefox to look?']` might be worth testing if this is worth testing though.